### ISSUE TITLE
kafka firehose reading from offset after engine restart throws exception

### ISSUE THREAD

**Issue by khou** [dashbase/dashbase-extensions#31]
```
INFO 2017-02-24 22:50:10,871 [Thread-15] (AbstractConnector.java:270): Started ServerConnector@549b99f0{HTTP/1.1,[http/1.1]}{0.0.0.0:7888}
INFO 2017-02-24 22:50:10,871 [Thread-15] (Server.java:397): Started @27131ms
INFO 2017-02-24 22:50:11,049 [rapid data processor] (DataEventProcessor.java:129): Resume reading from offset {"0":44482609}
ERROR 2017-02-24 22:50:11,053 [rapid data processor] (LifecycleModule.java:81): Uncaught exception from Thread[rapid data processor,5,main]:java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
	at io.dashbase.firehose.kafka_10.Kafka10Firehose.seekToOffset(Kafka10Firehose.java:139)
	at rapid.indexing.DataEventProcessor.run(DataEventProcessor.java:131)
	at java.lang.Thread.run(Thread.java:745)
INFO 2017-02-24 22:57:26,510 [Thread-10] (IndexPurgeThread.java:45): load segment list...
INFO 2017-02-24 22:57:26,518 [Thread-10] (IndexPurgeThread.java:47): Found 19 segments
```

**Comment by javasoze** [comment]
@khou I tried to following code snippet and it works:

    public static void main(String[] args) throws Exception {
	ObjectMapper mapper = new ObjectMapper();
	String str = "{\"0\":44482609}";
	Map<Integer, AtomicLong> val = mapper.readValue(str, Map.class);
	for (Entry<Integer, AtomicLong> entry : val.entrySet()) {
	  System.out.println(entry.getKey());
	  System.out.println(entry.getValue());
	}
    }

**Comment by javasoze** [comment]
A work around is to wrap it with a class

    public class KafkaOffset {
       public Map<Integer, AtomicLong> offsetMap = new HashMap<>();
    }

and now serialize this class works:

    public static void main(String[] args) throws Exception {
	  KafkaOffset offset = new KafkaOffset();
	  offset.offsetMap = ImmutableMap.of(1,new AtomicLong(2));
	ObjectMapper mapper = new ObjectMapper();
	String str = mapper.writeValueAsString(offset);
	System.out.println(str);
	offset = mapper.readValue(str, KafkaOffset.class);
	
       for (Entry<Integer, AtomicLong> entry : offset.offsetMap.entrySet()) {
		int key = entry.getKey();
		long v = entry.getValue().longValue();
		System.out.println(key + ":" + v);
        }
      }

**Comment by khou** [comment]
should be fixed in #30
